spec/src/main/asciidoc/changelog.adoc

```diff
@@ -19,75 +19,158 @@
+*** icon:bolt[role="red"] In OpenMetrics output format, the separator between scope and metric name is now a `_` instead of a `:`. (https://github.com/eclipse/microprofile-metrics/issues/279[#279])
+*** icon:bolt[role="red"] Metric names with camelCase are no longer converted to snake_case for OpenMetrics output. (https://github.com/eclipse/microprofile-metrics/issues/357[#357])
+** icon:bolt[role="red"] The default value of the `reusable` attribute for metric objects created programmatically (not via annotations) is now `true` (https://github.com/eclipse/microprofile-metrics/issues/328[#328])
+** icon:bolt[role="red"] Some base metrics' names have changed to follow the convention of ending the name of accumulating counters with `total`. (https://github.com/eclipse/microprofile-metrics/issues/375[#375])
+** icon:bolt[role="red"] Some base metrics' types have changed from Counter to Gauge since Counters must now count monotonically. (https://github.com/eclipse/microprofile-metrics/issues/375[#375])
+** icon:bolt[role="red"] Some base metrics' names have changed because they now use tags to distinguish metrics for multiple JVM objects. For example,
+each existing garbage collector now has its own `gc.total` metric with the name of the garbage collector being in a tag. Names 
+of some base metrics in the OpenMetrics output are also affected by the removal of conversion from camelCase to snake_case. (https://github.com/eclipse/microprofile-metrics/issues/375[#375])
+** Added a set of recommendations how application servers with multiple deployed applications should behave if they support MP Metrics. (https://github.com/eclipse/microprofile-metrics/issues/240[#240])
+
+
+[[release_notes_1_1]]
+== Changes in 1.1
+
+A full list of changes may be found on the link:https://github.com/eclipse/microprofile-metrics/milestone/1[MicroProfile Metrics 1.1 Milestone]
+
+=== API/SPI Changes
 ** `org.eclipse.microprofile.metrics.MetricRegistry.register(String name, Metric, Metadata)` is deprecated.
 Use `org.eclipse.microprofile.metrics.MetricRegistry.register(Metadata, Metric)` instead, where `Metadata`
 already has a field for the name.
-** Global tags are now supplied via the means of MicroProfile Config (the env variable is still valid).
+=== Functional Changes
+** `org.eclipse.microprofile.metrics.MetricRegistry.register(String name, Metric, Metadata)` is deprecated.
+Use `org.eclipse.microprofile.metrics.MetricRegistry.register(Metadata, Metric)` instead, where `Metadata`
+already has a field for the name.
+** Global tags are now supplied via the means of MicroProfile Config (the env variable is still valid). (https://github.com/eclipse/microprofile-metrics/issues/165[#165])
+=== Specification Changes
 ** Annotations and `Metadata` can now have a flag `reusable` that indicates that the metric name can be registered
 more than once. Default is `false` as in Metrics 1.0. See <<architecture#reusing_metrics>>.
+=== TCK enhancement
```

Comment:
This needs to be preceded by an empty line, otherwise it gets rendered as part of the previous line

Reply:
fixed